src/routes/Send.tsx

```diff
@@ -567,7 +567,8 @@ export function Send() {
                     lnurlp()!,
                     amountSats(),
                     undefined, // zap_npub
-                    tags
+                    tags,
+                    whatForInput().trim()
```

Comment:
Hmm. Nack IMO. We should be distinguishing between what is public and what is private.

Reply:
Good point, we can bring this back when we have the switcher